ACadSharp/IO/DXF/DxfStreamWriter/DxfObjectsSectionWriter.cs

```diff
@@ -37,7 +37,7 @@ protected void writeObject<T>(T co)
 					this.writeMappedObject<DictionaryVariable>(dictvar);
 					break;
 				case SortEntitiesTable sortensTable:
-					//this.writeSortentsTable(sortensTable);
+					this.writeSortentsTable(sortensTable);
```

Comment:
Same as the reader `writeSortentsTable(sortensTable)` has some errors, please do not uncomment this line.